compiler/rustc_hir_analysis/src/astconv/mod.rs

```diff
@@ -1181,10 +1181,7 @@ impl<'o, 'tcx> dyn AstConv<'tcx> + 'o {
                 if let Some(bound_span) = bound_span {
                     err.span_label(
                         bound_span,
-                        format!(
-                            "ambiguous `{assoc_name}` from `{}`",
-                            bound.print_only_trait_path(),
-                        ),
+                        format!("ambiguous `{assoc_name}` from `{}`", bound.print_trait_sugared(),),
```

Comment:
```suggestion
                        format!("ambiguous `{assoc_name}` from `{}`", bound.print_trait_sugared()),
```

Reply:
We have a lot of these :(

I ctrl+f'd for `,);` and `,),` yesterday and found like >50 instances.